examples/examples.py

```diff
@@ -6,101 +8,54 @@
-)
-input_text = "I love this product!"
-response = client.request(SentimentAnalysisRequest(text=input_text))
-print_response("AWS", Task.SENTIMENT_ANALYSIS, input_text, response)
-
-# Sending a Translation Request to Azure
-client = supercontrast_client(
-    task=Task.TRANSLATION,
-    providers=[Provider.AZURE],
-    optimizer=None,
-    source_language="en",
-    target_language="fr",
-)
-input_text = "I love this product!"
-response = client.request(TranslationRequest(text=input_text))
-print_response("Azure", Task.TRANSLATION, input_text, response)
-
-# Sending a OCR Request to Sentisight
-client = supercontrast_client(
-    task=Task.OCR,
-    providers=[Provider.SENTISIGHT],
-    optimizer=None,
-    language="en",
-)
-input_image = "https://jeroen.github.io/images/testocr.png"
-response = client.request(OCRRequest(image=input_image))
-print_response("Sentisight", Task.OCR, input_image, response)
-
-# Sending a OCR Request to API4AI
-client = supercontrast_client(
-    task=Task.OCR,
-    providers=[Provider.API4AI],
-    optimizer=None,
-    language="en",
-)
-input_image = "https://jeroen.github.io/images/testocr.png"
-response = client.request(OCRRequest(image=input_image))
-print_response("API4AI", Task.OCR, input_image, response)
-
-client = supercontrast_client(
-    task=Task.OCR,
-    providers=[Provider.CLARIFAI],
-    optimizer=None,
-    language="en",
-)
-input_image = "https://jeroen.github.io/images/testocr.png"
-request = OCRRequest(image=input_image)
-response = client.request(request)
-print_response("Clarifai", Task.OCR, input_image, response)
-
-
-client = supercontrast_client(
-    task=Task.TRANSLATION,
-    providers=[Provider.MODERNMT],
-    optimizer=None,
-    source_language="en",
-    target_language="fr",
-)
-input_text = "I love this product!"
-response = client.request(TranslationRequest(text=input_text))
-print_response("ModernMT", Task.TRANSLATION, input_text, response)
-
-input_audio = "https://github.com/voxserv/audio_quality_testing_samples/raw/master/mono_44100/127389__acclivity__thetimehascome.wav"
-
-client = supercontrast_client(
-    task=Task.TRANSCRIPTION,
-    providers=[Provider.OPENAI],
-    optimizer=None,
-)
-
-response = client.request(TranscriptionRequest(audio_file=input_audio))
-print_response("OpenAI", Task.TRANSCRIPTION, input_audio, response)
-
-client = supercontrast_client(
-    task=Task.TRANSCRIPTION,
-    providers=[Provider.AZURE],
-    optimizer=None,
-)
-response = client.request(TranscriptionRequest(audio_file=input_audio))
-print_response("Azure", Task.TRANSCRIPTION, input_audio, response)
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+
+def create_client(task, provider, **kwargs):
+    """Create a supercontrast client for the given task and provider."""
+    return supercontrast_client(task=task, providers=[provider], **kwargs)
+
+async def process_request(client, request, provider, task, input_data):
```

Comment:
this is a great suggestion, and we definitely plan on supporting async in the future, but we will probably be doing that inside of the package, rather than in the examples file